frontend/public/components/pod-exec.jsx

```diff
@@ -74,7 +74,7 @@ export class PodExec extends React.PureComponent {
         if (raw[0] === '3') {
           if (previous.includes(NO_SH)) {
             current.reset();
-            current.onConnectionClosed(`This container doesn't have a /bin/sh shell. Try specifying your command in a terminal with:\r\n\r\n  kubectl -n ${metadata.namespace} exec ${metadata.name} -ti <command>`);
+            current.onConnectionClosed(`This container doesn't have a /bin/sh shell. Try specifying your command in a terminal with:\r\n\r\n  oc -n ${metadata.namespace} exec ${metadata.name} -ti <command>`);
```

Comment:
Console is intended to work against upstream Kubernetes, not just OpenShift. We can use `connectToFlags()` to detect if we are running on OpenShift and display the appropriate message.